android/app/build.gradle

```diff
@@ -50,6 +50,11 @@ android {
         exclude 'META-INF/maven/commons-io/commons-io/pom.properties'
         exclude("META-INF/*.kotlin_module")
         exclude 'README.txt'
+
+        jniLibs {
+            useLegacyPackaging true
```

Comment:
This is the most important change. For whatever reason, using legacy packaging reduces the size of the APK by 100MB!

Reply:
wooo, That's great.